examples/create-react-app-with-typescript/src/pages/MuxUploader.tsx

```diff
@@ -24,7 +24,6 @@ function MuxUploaderPage() {
         style={{ display: "block", height: "200px" }}
         endpoint={url}
         type="bar"
```

Comment:
nitpick(blocking): type is no longer a functional property on <MuxUploader/>. Remove (and all other cases).